packages/core-database-postgres/src/repositories/repository.ts

```diff
@@ -103,8 +103,8 @@ export abstract class Repository implements Database.IRepository {
             return { rows, count: Math.max(count, rows.length), countIsEstimate: true };
         }
 
-        const countRow = await this.find(selectQueryCount);
+        const { count } = await this.db.one(`SELECT COUNT(*) FROM (${selectQueryCount.toString()}) AS count;`);
```

Comment:
If you are going to do `SELECT COUNT(*) FROM (... subquery ...)` then you do not need a separate `selectQueryCount` variable and can use `selectQuery` for this. But I specifically did not do that when I introduced `selectQueryCount` because the straight-forward `SELECT COUNT(*) FROM table WHERE ...` is to be preferred over the convoluted `SELECT COUNT(*) FROM (SELECT ... FROM table WHERE ...) AS count;` because with the latter you are asking the database to do more stuff and depend on it being smart enough to detect that both queries are the same and actually do the first one under the hood. In other words, the variant with the subquery could also be slower!